src/Microsoft.Data.Sqlite.Core/SqliteDataReader.cs

```diff
@@ -21,29 +16,26 @@ namespace Microsoft.Data.Sqlite
     /// </summary>
     public class SqliteDataReader : DbDataReader
     {
-        // TODO can be Array.Empty<T>() when upgrading to net46
```

Comment:
Only if we decide to drop support for older platforms which I'm not sure we should with this library...